src/status_im/common/metrics_confirmation_modal/view.cljs

```diff
@@ -0,0 +1,74 @@
+(ns status-im.common.metrics-confirmation-modal.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im.common.metrics-confirmation-modal.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- hide-bottom-sheet
+  []
+  (rf/dispatch [:hide-bottom-sheet]))
+
+(defn- toggle-metrics
+  [enabled?]
+  (rf/dispatch [:centralized-metrics/toggle-centralized-metrics enabled?]))
+
+(def will-receive-points
+  [(i18n/label :t/ip-address)
+   (i18n/label :t/universally-unique-identifiers-of-device)
+   (i18n/label :t/logs-of-actions-withing-the-app)])
+
+(def not-receive-points
+  [(i18n/label :t/your-profile-information)
+   (i18n/label :t/your-addresses)
+   (i18n/label :t/information-you-input-and-send)])
+
+(defn- render-points
+  [{:keys [title points]}]
+  [rn/view
+   [quo/text {:weight :semi-bold}
+    title]
+   (map-indexed
```

Comment:
This doesn't work because it's returning a sequence, but `rn/view` doesn't know what to do with it.

<img src="https://github.com/user-attachments/assets/fc0b1819-4f4d-47a6-ae45-3b2acb56657f" width="300" />

Reply:
Also from the screenshot it appears that the input has focus and that's why the keyboard is shown. I was confused for a moment because I thought I had to type something to accept the conditions.

A minor detail, but we can force hide the keyboard if this bottom sheet appears.

Reply:
interesting, it works locally, maybe on dev mode it realizes it